docker/start.sh

```
@@ -0,0 +1,94 @@
+  lightcyan="\033[1;36m"
+  nocolor="\033[0m"
+  echo -e "\n${lightcyan}$1${nocolor}\n"
+}
+
+# Let the agent ignore the token env variables
+export VSO_AGENT_IGNORE="AZP_TOKEN,AZP_TOKEN_FILE"
+
+print_header "1. Determining matching Azure Pipelines agent..."
+
+AZP_AGENT_PACKAGES=$(curl -LsS \
+    -u user:$(cat "${AZP_TOKEN_FILE}") \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)